filament/src/details/ColorGrading.cpp

```diff
@@ -581,8 +580,11 @@ static float3 luminanceScaling(float3 x,
 // Quality
 //------------------------------------------------------------------------------
 
-static std::tuple<TextureFormat, PixelDataFormat, PixelDataType>
-        selectLutTextureParams(ColorGrading::LutFormat const lutFormat) noexcept {
+static std::tuple<TextureFormat, PixelDataFormat, PixelDataType> selectLutTextureParams(
+        ColorGrading::LutFormat const lutFormat, const bool isOneDimensional) noexcept {
+    if (isOneDimensional) {
+        return { TextureFormat::R8, PixelDataFormat::R, PixelDataType::UBYTE };
```

Comment:
Why not use a fp16 texture? 8 bits is generally not enough to encode the sRGB<>Linear transform.

Reply:
Done.